services/apiService.ts

```diff
@@ -112,3 +112,12 @@ export const getPendingBoostClaims = async (addr: string) => {
     console.log(err);
```

Comment:
Same comment that on the other PR.

console.log("Error while ...", error);

Reply:
done for others in boost v2 PR